src/cli/index.ts

```diff
@@ -92,12 +93,16 @@ async function handleError(args, error) {
   spinner.clearAll();
   let command = 'bad-command';
   let exitCode = EXIT_CODES.ERROR;
+
   const noSupportedManifestsFound = error.message?.includes(
     'Could not detect supported target files in',
   );
+  const noSupportedSastFiles = error instanceof NoSupportedSastFiles;
+  const noSupportedProjectsDetected =
+    noSupportedManifestsFound || noSupportedSastFiles;
```

Comment:
Shouldn't this be an `and` condition?
```suggestion
  const noSupportedProjectsDetected =
    noSupportedManifestsFound && noSupportedSastFiles;
```

Reply:
Nope, we would like to exit with code `3` (NO_SUPPORTED_PROJECTS_DETECTED) when we don't have supported files detected. and we can reach this state with one of those checks